src/java/htsjdk/tribble/readers/LineReaderUtil.java

```diff
@@ -1,94 +1,53 @@
 package htsjdk.tribble.readers;
 
-import htsjdk.samtools.Defaults;
-import htsjdk.samtools.util.CloserUtil;
-import htsjdk.samtools.util.RuntimeIOException;
-import htsjdk.tribble.TribbleException;
-
-import java.io.IOException;
 import java.io.InputStream;
-import java.io.InputStreamReader;
 import java.io.StringReader;
 
 /**
  * A collection of factories for generating {@link LineReader}s.
  *
+ * @Deprecated use {@link SynchronousLineReader} directly.
  * @author mccowan
  */
+@Deprecated
 public class LineReaderUtil {
+    @Deprecated
     public enum LineReaderOption {
-        ASYNCHRONOUS, SYNCHRONOUS
+        ASYNCHRONOUS,
+        SYNCHRONOUS
     }
 
     /**
-     * Like {@link #fromBufferedStream(java.io.InputStream, LineReaderUtil.LineReaderOption)}, but the synchronicity
-     * option is determined by {@link htsjdk.samtools.Defaults}: if asynchronous I/O is enabled, an asynchronous line reader will be
-     * returned.
+     * @Deprecated use <code>new SynchronousLineReader(stream);</code>
      */
+    @Deprecated
     public static LineReader fromBufferedStream(final InputStream stream) {
-        return fromBufferedStream(stream, Defaults.USE_ASYNC_IO_FOR_TRIBBLE ? LineReaderOption.ASYNCHRONOUS : LineReaderOption.SYNCHRONOUS);
+        return new SynchronousLineReader(stream);
```

Comment:
With these changes, what is `Defaults.USE_ASYNC_IO_FOR_TRIBBLE` still hooked up to, if anything?